buildscripts/build-managed.cmd

```diff
@@ -40,21 +40,24 @@ set "NUGET_PACKAGES=%__PackagesDir%"
 echo Using CLI tools version:
 dir /b "%__DotNetCliPath%\sdk"
 
-"%__DotNetCliPath%\dotnet.exe" msbuild "%__ProjectDir%\build.proj" /nologo /t:Restore /flp:v=normal;LogFile=build-restore.log /p:NuPkgRid=win7-x64 /maxcpucount /p:OSGroup=%__BuildOS% /p:Configuration=%__BuildType% /p:Platform=%__BuildArch% %__ExtraMsBuildParams%
+set __rid=win7-x64
```

Comment:
It would be nice to set this in buildvars-setup.cmd and call it `__NugetRuntimeId` - it is how it is done in the Unix shell scripts.

Reply:
Changed to `__NugetRuntimeId`.